lib/adoptoposs/submissions.ex

```diff
@@ -208,7 +209,8 @@ defmodule Adoptoposs.Submissions do
             "SELECT COUNT(*) FROM interests WHERE project_id = ? AND creator_id = ?",
             p.id,
             ^user_id
-          ) == 0
+          ) == 0,
+        order_by: [desc: p.id]
```

Comment:
since we have `timestamps()` in `project.ex`, could we not just sort by creation date? I mean `id` will keep stable, as long we use numerical IDs, but sorting by timestamps might give other options, e.g. sorting by the last project updated, etc.

Reply:
:+1: Good point, will change it to use `inserted_at`.